lib/elixir/lib/module/checker.ex

```diff
@@ -42,9 +42,9 @@ defmodule Module.Checker do
   end
 
   # Mod.fun(...)
-  defp check_body({{:., meta, [module, fun]}, _, args}, state)
+  defp check_body({{:., meta, [module, fun]}, call_meta, args}, state)
        when is_atom(module) and is_atom(fun) do
-    check_remote(module, fun, length(args), meta, state)
+    check_remote(module, fun, length(args), meta ++ call_meta, state)
```

Comment:
This happens because expansion of imports do not add the metadata in all nodes, we need to the same above for `&Mod.fun/arity`. I am not sure if this is expected or we should fix it in the expansion pass?